src/EditorFeatures/CSharpTest/CodeRefactorings/UseRecursivePatterns/UseRecursivePatternsRefactoringTests.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
```

Comment:
Should be easy to remove and enable NRT.

Reply:
This is a test file?

Reply:
@alrz I usually don't add `#nullable disable` for new files (whether it's a test or a product code).

Reply:
yeah, we don't need this for test files.  and most test files can easily strip this without needing any other changes.  i just haven't gotten around to doing this in bulk yet.